packages/main/src/Table.ts

```diff
@@ -564,6 +601,10 @@ class Table extends UI5Element {
 	get isTable() {
 		return true;
 	}
+
+	get dropIndicatorDOM(): DropIndicator | null {
+		return this.shadowRoot!.querySelector("[ui5-drop-indicator]");
```

Comment:
how this can be null maybe you should tell TS it cannot be null and remove ! from all this.dropIndicatorDOM! access